homeassistant/components/overkiz/cover_entities/vertical_cover.py

```diff
@@ -64,7 +65,7 @@ def supported_features(self) -> CoverEntityFeature:
 
         return supported_features
 
-    @property
+    @cached_property
     def device_class(self) -> CoverDeviceClass:
         """Return the class of the device."""
         return (
```

Comment:
This can be set in the constructor in a separate PR as well (but will need one)

Reply:
#100394